load.js

```
@@ -46,17 +55,44 @@ function mergeDeep(target, source) {
 function canMerge(x, y) { return (isObject(x) && isObject(y)) || (isArray(x) && isArray(y)) }
+/**
+ * @arg {any} x
+ * @returns {x is Record<string, any> | Array<any>}
+ */
 function shouldClone(x) { return isObject(x) || isArray(x) }
+/**
+ * @arg {any} x
+ * @returns {x is Record<string, any>}
+ */
 function isObject(x) {
   const constructor = getConstructor(x)
```

Biome (2.1.2)

[error] Do not shadow the global "constructor" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)